auth/api/grpc/server.go

```diff
@@ -423,7 +423,7 @@ func decodeListObjectsRequest(_ context.Context, grpcReq interface{}) (interface
 		ObjectType:    req.GetObjectType(),
 		Object:        req.GetObject(),
 		NextPageToken: req.GetNextPageToken(),
-		Limit:         req.GetLimit(),
+		Limit:         uint64(req.GetLimit()),
```

Comment:
Change the `limit` in the proto file to `uint64` too.

Reply:
It raises an issue during the test hence it doesn't pass it

Reply:
Please fix the tests as well, in that case.